ESPixelStick/ESPixelStick.ino

```diff
@@ -76,25 +59,37 @@ static void _u0_putc(char c){
 //  Globals
 //
 /////////////////////////////////////////////////////////
+// OLED
```

Comment:
Please create an OLED file and move all of this to that file. Please make the entire file contents conditional (look at the temperature sensor setup). Then enable this in your platform.